site/src/_layouts/default.hbs

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <title>d3fc {{#if title}}:{{/if}} {{title}}</title>
+    <meta name="description" content="A collection of components that make it easy to build interactive charts with D3.">
+
+    <script src="{{assets}}/scripts.js"></script>
+
+    <link rel="stylesheet" href="{{assets}}/styles.css" />
+  </head>
+  <body>
+    <div class="content">
+
+      <nav class="navbar navbar-inverse">
+        <div class="container">
+            <div class="navbar-header">
+              <a href="{{assets}}/"><img class="navbar-logo" src="{{assets}}/images/logo.svg" /></a>
```

Comment:
My 2¢ - this is a very conventional layout, people will know that this is a link from its location, so i don't think we really need a hover effect,